threat_intel/opendns.py

```diff
@@ -265,3 +270,8 @@ def latest_malicious(self, ips):
         api_name = 'opendns-latest_malicious'
         fmt_url_path = u'ips/{0}/latest_domains'
         return self._multi_get(api_name, fmt_url_path, ips)
+
+
+class ResponseError(Exception):
+
+    """Raised when the response received from the endpoint is not valid."""
```

Comment:
So this is actually code that will run for the uncached domains. I will add the error message in the exception.